srml/contract/src/gas.rs

```diff
@@ -204,14 +205,14 @@ pub fn buy_gas<T: Trait>(
 ) -> Result<(GasMeter<T>, NegativeImbalanceOf<T>), &'static str> {
 	// Check if the specified amount of gas is available in the current block.
 	// This cannot underflow since `gas_spent` is never greater than `block_gas_limit`.
```

Comment:
If we want to apply the change a line below then the comment should be updated

```suggestion
	// This cannot underflow since `gas_spent` is never greater than `T::BlockGasLimit`.
```